site/src/pages/_app.page.tsx

```diff
@@ -9,11 +9,85 @@ import { IntlProvider } from "react-intl";
 import { createGlobalStyle } from "styled-components";
 
 const GlobalStyle = createGlobalStyle`
+    /* this fixes a problem with flexbox and avoids overflows https://defensivecss.dev/tip/flexbox-min-content-size/ */
+    * {
+        min-width: 0;
+    }
+
+    /*
+        Prevent adjustments of font size after orientation changes in iOS.
```

Comment:
```suggestion
        Prevent adjustments of font size after orientation changes in iOS
```

Nit: Either use proper punctuation everywhere, or don't use it at all 😁